forge-gui/res/cardsfolder/upcoming/final_flourish.txt

```diff
@@ -0,0 +1,8 @@
+Name:Final Flourish
+ManaCost:1 B
+Types:Instant
+K:Kicker:Sac<1/Creature;Artifact/creature or artifact>
```

Comment:
`/artifact or creature>` to match the oracle

Reply:
Is Forge building the reminder text well for this kicker in the Card Detail? Not something you'd need to fix in this PR, just curious.